src/components/services/Rtorrent.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <span v-if="error" class="error">An error has occurred.</span>
+        <template v-else>
+          <span class="down">
+            <i class="fas fa-download"></i> {{ downRate }}
+          </span>
+          <span class="up">
+            <i class="fas fa-upload"></i> {{ upRate }}
+            </span>
+        </template>
+      </p>
+    </template>
+    <template #indicator>
+      <span v-if="!error" class="count">{{ count }}
+        <template v-if="count === 1">torrent</template>
+        <template v-else>torrents</template>
+      </span>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import Generic from './Generic.vue';
+
+// Units to add to download and upload rates.
+const units = ['B', 'kiB', 'MiB', 'GiB'];
+
+// Take the rate in bytes and keep dividing it by 1k until the lowest
+// value for which we have a unit is determined. Return the value with
+// up to two decimals as a string and unit/s appended.
+const displayRate = (rate) => {
+  let i = 0;
+  
+  while (rate > 1000 && i < units.length) {
+    rate /= 1000;
+    i++;
+  }
+
+  return Intl.NumberFormat(undefined, {maximumFractionDigits: 2})
+    .format(rate || 0) + ` ${units[i]}/s`;
+}
+
+export default {
+  name: 'rTorrent',
+  props: {item: Object},
+  components: {Generic},
+  // Properties for download, upload, torrent count and errors.
+  data: () => ({dl: null, ul: null, count: null, error: null}),
+  // Computed properties for the rate labels.
+  computed: {
+    downRate: function() {
+      return displayRate(this.dl);
+    },
+    upRate: function() {
+      return displayRate(this.ul);
+    },
+  },
+  created() {
+    // Set intervals if configured so the rates and/or torrent count
+    // will be updated.
+    const rateInterval = parseInt(this.item.rateInterval, 10) || 0;
+    const torrentInterval = parseInt(this.item.torrentInterval, 10) || 0;
+
+    if (rateInterval > 0) {
+      setInterval(() => this.fetchRates(), rateInterval);
+    }
+
+    if (torrentInterval > 0) {
+      setInterval(() => this.fetchCount(), torrentInterval);
+    }
+    
+    // Fetch the initial values.
+    this.fetchRates();
+    this.fetchCount();
+  },
+  methods: {
+    // Perform two calls to the XML-RPC service and fetch download
+    // and upload rates. Values are saved to the `ul` and `dl`
+    // properties.
+    fetchRates: async function() {
+      try {
+        await this.getRate('throttle.global_up.rate')
+          .then((ul) => this.ul = ul);
+        await this.getRate('throttle.global_down.rate')
+          .then((dl) => this.dl = dl);
+      } catch {
+        this.error = true;
+      }
```

Comment:
Sorry for not pointing that out in my first review but instead of waiting for the first request to finish to execute the second one (like your first implementation with the chain), why not run both request in parallel for faster execution? Something like (untested): 
```suggestion
       this.getRate('throttle.global_up.rate')
         .then((ul) => this.ul = ul)
         .catch(() => this.error = true); 
       this.getRate('throttle.global_down.rate')
         .then((dl) => this.dl = dl);
         .catch(() => this.error = true); 
```